source/packages/integration-tests/src/di/inversify.config.ts

```diff
@@ -21,11 +21,14 @@ import {provisioningContainerModule} from '@cdf/provisioning-client';
 import {notificationsContainerModule} from '@cdf/notifications-client';
 import {greengrassDeploymentContainerModule} from '@cdf/greengrass-deployment-client';
 import {greengrassProvisioningContainerModule} from '@cdf/greengrass-provisioning-client';
-
+import { CDFConfigInjector } from '@cdf/config-inject';
 
 // Load everything needed to the Container
 export const container = new Container();
 
+const configInjector = new CDFConfigInjector();
```

Comment:
HI @kaerukoara , your PR looks good, but for now could you remove this reference to configInjector, I believe you use it because you want to run your test in both lambda and api gateway mode to test your changes.

But at the moment, to keep things consistent all of our acceptance tests are being invoked through API Gateway, so we don't need this for now (we might have a task in the future to properly implement the tests to be run in lambda and api gateway mode). 

Once you do that, I'll approve and merge your changes.

Thanks again

Reply:
Hi, @williamsia, I see.   I have restored my changes under your suggestion. Thanks for your confirmation.